src/app/shared/markdown-wrapper/markdown-wrapper.service.ts

```diff
@@ -109,4 +110,18 @@ export class MarkdownWrapperService {
   removeTabsFromTableHeaders(data: string): string {
     return data.replace(/(^ {0,3}\|.*)/gm, (match) => match.replace(/\t/g, '    '));
   }
+
+  /**
+   * Add the query 'raw=true' to all img href urls that refer to github, causing github to redirect to the raw image.
+   */
+  makeGitHubImagesRaw(html: string): string {
```

Comment:
This would be a good candidate for some unit tests, like there are for `removeTabsFromTableHeaders`, especially since regular expressions are hard, for me at least, to figure out the details of what they're doing without going back to a reference manual.